Wikipedia/Code/WMFNearbyContentSource.m

```diff
@@ -59,20 +63,35 @@ - (WMFLocationSearchFetcher *)locationSearchFetcher {
 
 - (void)startUpdating {
     self.isFetchingInitialLocation = NO;
-    [self.currentLocationManager startMonitoringLocation];
+    if ([WMFLocationManager isAuthorized]) {
+        [self.currentLocationManager startMonitoringLocation];
+    }
 }
 
 - (void)stopUpdating {
-    [self.currentLocationManager stopMonitoringLocation];
+    if ([WMFLocationManager isAuthorized]) {
```

Comment:
Looks like stopMonitoringLocation will no-op if it's not actually updating, so it is safe to always call it regardless of auth status. Will update.